README.md

```diff
@@ -17,13 +18,40 @@
 * Operates on top of Selenium.
 * The Git for your GUI.
 
+
+### Easy creation and maintenance
+
+Instead of manually defining individual aspects that you want to check, check everything at once. 
+So instead of writing lots of `assert`-statements (and still not have complete checks), write a single `re.check`.
+This saves a lot of effort when creating the tests. 
+And makes sure to not [miss unexpected changes](https://hackernoon.com/assertions-considered-harmful-d3770d818054).
+
+And even better: using the [retest GUI](https://retest.de/en/) (or the soon to come open source CLI), you can easily accept those changes with a single click (patent pending).
+This also saves a lot of time during maintenance.
+Any regular changing aspects or elements can easily be ignored.
+
+
 ### Prerequisites
 
 Currently only available as a Java API with support for JUnit 4.
 
+
 ### Installing
 
-We will soon provide a bundled JAR and Maven dependencies.
+You can download recheck-web [directly](https://github.com/retest/recheck-web/releases/) or use Maven and have the transitive dependencies be automatically resolved by adding it as a Maven dependency in your POM:
+
+```
+<dependency>
+	<groupId>de.retest</groupId>
+	<artifactId>recheck-web</artifactId>
+	<version>0.1.0</version>
+</dependency>
+```
+
+
+## License
```

Comment:
Usually, license is at the bottom since GitHub already shows the license (via `LICENSE`) above.

Reply:
Also, this looks like "Building" (h3) is a subsection of "License" (h2).